schema/filter_artworks.js

```diff
@@ -186,6 +216,7 @@ function filterArtworks(primaryKey) {
       }
 
       return gravity.with(accessToken)("filter/artworks", gravityOptions)
+        .then(response => assign({}, response, { options: gravityOptions }))
```

Comment:
This could probably do with a test to ensure the intent is clear to ensure future devs don't accidentally introduce regressions.